client/lib/gsuite/get-google-mail-service-family.js

```diff
@@ -19,6 +20,7 @@ export function getGoogleMailServiceFamily( productSlug = null ) {
 			case GSUITE_BUSINESS_SLUG:
 				return GSUITE_PRODUCT_FAMILY;
 
+			case GOOGLE_WORKSPACE_BUSINESS_STARTER_MONTHLY:
```

Comment:
Seems like we can exchange the `switch` statement for readable `if` blocks. We can take advantage of the `isGoogleWorkspaceProductSlug` method here.